sdk/containerregistry/azure-containerregistry/azure/containerregistry/_registry_artifact.py

```diff
@@ -30,15 +28,17 @@
     from ._models import ContentProperties
 
 
-class ContainerRepositoryClient(ContainerRegistryBaseClient):
-    def __init__(self, endpoint, repository, credential, **kwargs):
-        # type: (str, str, TokenCredential, Dict[str, Any]) -> None
-        """Create a ContainerRepositoryClient from an endpoint, repository name, and credential
+class RegistryArtifact(ContainerRegistryBaseClient):
+    def __init__(self, endpoint, repository, tag_or_digest, credential, **kwargs):
```

Comment:
It looks like this is also kind of a client - but it didn't have the pipeline/transport wrapping in the methods that construct it.

Reply:
The reasoning is the same as for `ContainerRegistry`, but I do need to add the pipeline/transport wrapping constructs.